README.md

```diff
@@ -1,11 +1,28 @@
 # Kaffe
 
-An opinionated, highly specific, Elixir wrapper around
-[Brod](https://github.com/klarna/brod): the Erlang Kafka client.
-:coffee:
+An opinionated, highly specific, Elixir wrapper around [Brod](https://github.com/klarna/brod): the Erlang Kafka client. :coffee:
```

Comment:
Most of the changes in the README, besides adding the table of contents, are just unwrapping lines.